tox.ini

```
@@ -0,0 +1,15 @@
+[tox]
+requires =
+    tox>=4.2
+env_list =
+    py311
+    py310
+    py38
+
+[testenv]
+commands =
+    pytest --cov betterproto
```

```suggestion
    pytest {posargs:--cov betterproto}
```

This is _really_ useful to override the arguments passed to `pytest`. It means you can run just one test with:

```
tox -e py -- betterproto/test_abc.py
```